lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -135,21 +135,17 @@ namespace Js
             // Clear ReentrancyLock bit as initialization code doesn't have any side effect
             scriptContext->GetThreadContext()->SetNoJsReentrancy(false);
 #endif
+            // specify which set of BuiltIns are currently being loaded
```

Comment:
Changes in this file are a ride-along refactor, with no functional impact.

I did many refactors whilst trying to fix this bug and wanted to keep this one.